server.js

```diff
@@ -16,15 +16,30 @@ app.use(cors())
 // return location info
 app.get('/location', (request, response) => {
     const locationData = require('./data/location.json')
-    response.send(new Location(locationData))
+    response.send(new Location(locationData[0], request.query.city))
 });
 
+function Location(obj, city) {
+    this.search_query = city,
+    this.formatted_query = obj.display_name,
+    this.latitude = obj.lat,
+    this.longitude = obj.lon
+}
+
+//return weather info
+app.get('/weather', (request, response) => {
+    const weatherData = require('./data/weather.json')
```

Comment:
i like that you used full words for the parameters, i had no idea what they were for so long